modules/objc/examples/code_snippets/GettingStarted.m

```diff
@@ -13,28 +22,31 @@ @implementation GettingStarted
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
-    // Get the database (and create it if it doesn’t exist).
-
-    bool *testReplication;
-    testReplication = false;
+    
+    [self getStartedWithReplication: false];
+}
 
+- (void) getStartedWithReplication: (Boolean) replication {
```

Comment:
Is there a place in the doc that will call this with true or false?  It depends on the purpose of this big code snippet but I don't think we need this boolean flag at all?

Reply:
Yes, in `viewDidLoad`. The purpose of this is a longer discussion, but the main idea is that Repl with error on start due to the URL being dummy value if not changed. On top, there was no mention for iOS that SGW is needed. Added to that, all platforms have code comment that replication is optional, which is not reflected in the "implementation". I believe the main purpose of a such called `GettingStarted` is to verify that the CBL installation can actually run some basic operations. As we are going to have a lot more new developers, I believe this code might be copy-pasted-run so it should error/crash.

I've asked Blake to make similar changes https://github.com/couchbase/docs-couchbase-lite/pull/921 . For .NET and C, as we have validation pipeline in place, I thought I will give a try by myself and have you and Jim review afterwards (only for 3.2 docs).